src/pages/journal/journal.ts

```diff
@@ -119,11 +119,13 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
   };
 
   private createSlots = (emission: any) => {
-    if (!Array.isArray(emission) || emission.length === 0) {
-      return;
-    }
+    if (!Array.isArray(emission)) return;
```

Comment:
I like this change to the subscription conceptually. Personally not a fan of inlining the return, but I suppose the style standard is one for the wider team.

Reply:
I like inline returns, its my guilty JS pleasure